crates/nu-cli/src/commands/classified/external.rs

```diff
@@ -380,10 +380,13 @@ pub fn did_find_command(#[allow(unused)] name: &str) -> bool {
         if which::which(name).is_ok() {
             true
         } else {
+            // Reference: https://ss64.com/nt/syntax-internal.html
             let cmd_builtins = [
-                "call", "cls", "color", "date", "dir", "echo", "find", "hostname", "pause",
-                "start", "time", "title", "ver", "copy", "mkdir", "rename", "rd", "rmdir", "type",
-                "mklink",
+                "assoc", "break", "call", "cd", "chdir", "cls", "color", "copy", "date", "del",
+                "dir", "dpath", "echo", "endlocal", "erase", "exit", "for", "ftype", "goto", "if",
```

Comment:
I'm not sure all these additions make sense for Nu. For example `cd` isn't going to actually change the directory, `exit` won't actually exit, `rem` doesn't really have a use.

If there are cmd features we should add that give Nu more functionality, yeah let's definitely do it.

Reply:
Then I guess I don't understand what is going on here. Those cmmands I put are built into cmd.exe. If you want to run a batch file (*.bat; *.cmd) you'll need to be able to use those commands. Also, since `date` was already in there and we have our own date, I figured it was ok, to just be a complete-ist and put everything built-in in this code.

As it stands right now, batch files doesn't even work. I found this "incomplete" section of code while on my way to debug what batch files don't work. Which I can't debug now anyway because `thread 'main' has overflowed its stack`. But that's a separate issue.

Reply:
Right, I'm saying that we can extend the list, just we should add all of them.

For example:

```
> ^exit
```

Isn't going to do anything interesting. `^cd` I'm not sure will work, though it's worth a test I guess.

I do think most of the new things you're adding are probably okay, I just wanted to call out we should check them to make sure they do something useful and don't confuse users.

Reply:
Understood. I tested out all these commands and removed a bunch. I was stuck on running batch files and them not having what they needed however, if we do run batch files, I figure we'd shell out to cmd.exe somehow and run them and therefore cmd.exe would provide whatever we need.  What remains here I think is useful for either "real" commands or "diagnostic" commands. Let me know what you think.

Reply:
Looks like i removed some that were in there already.
* `call` is for batch files. 
* `^cls` does something but i'm not sure why we need it, unless it's useful for diagnostics
* `hostname` is an external command. 
* `pause` is for batch files.